tip-wlan/charts/opensync-gw-cloud/templates/service.yaml

```diff
@@ -5,35 +5,49 @@ metadata:
   namespace: {{ include "common.namespace" . }}
   labels:
     {{- include "common.labels" . | nindent 4 }}
+  {{- with .Values.service.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
 spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port1 }}
       targetPort: {{ .Values.service.port1 }}
       protocol: TCP
       name: {{ .Values.service.name1 }}
+      {{- if and .Values.service.nodePort_static (eq .Values.service.type "NodePort") }}
```

Comment:
Please align with other defined variables and use camelcase here: nodePortStatic would be better.